locations/spiders/roche_bobois.py

```
@@ -0,0 +1,42 @@
+from scrapy.spiders import SitemapSpider
+
+from locations.categories import apply_category, Categories
+from locations.hours import OpeningHours
+from locations.items import Feature
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class RocheBoboisSpider(SitemapSpider, StructuredDataSpider):
+    name = "roche_bobois"
+    item_attributes = {"brand": "Roche Bobois", "brand_wikidata": "Q3437504"}
+    sitemap_urls = ["https://www.roche-bobois.com/en-GB/sitemap_index.xml"]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)